service/authorization/authorization.proto

```diff
@@ -174,6 +188,7 @@ message EntityEntitlements {
 
 //A logical bucket of attributes belonging to a "Resource"
 message ResourceAttribute {
+  string resource_attributes_id = 1;
```

Comment:
Is this id ephemeral and not long-lasting? Is it possible to rely on an index instead of an on-the-fly id if its purpose is to track decision responses by the `repeated ResourceAttribute` within the token decision request?

Reply:
probably, i can break this off into a separate issue since we do the same thing for entities when getting multiple of them
also this was just a quick fix, the field was supposed to be there and is referenced in the docs but must have gotten removed

Reply:
https://github.com/opentdf/platform/issues/814